File: TOMIMSV4/Entity/TMS/clsBeTms_ticket_pol.vb
Public Class clsBeTms_ticket_pol
    Implements ICloneable

    Public Property IdTicket() As Integer = 0
    Public Property IdOrdenTmsEnc() As Integer = 0
    Public Property NoPoliza() As String = ""
    Public Property Dua() As String = ""
    Public Property Fecha_poliza() As Date = Date.Now
    Public Property Pais_procede() As String = ""
    Public Property Tipo_cambio() As Double = 0.0
    Public Property Total_valoraduana() As Double = 0.0
    Public Property Total_bultos_peso() As Double = 0.0
    Public Property Total_usd() As Double = 0.0
    Public Property Total_flete() As Double = 0.0
    Public Property Total_seguro() As Double = 0.0
    Public Property User_agr() As String = ""
    Public Property Fec_agr() As Date = Date.Now
    Public Property User_mod() As String = ""
    Public Property Fec_mod() As Date = Date.Now
    Public Property IdRegimen() As Integer = 0
    Public Property Clave_aduana() As String = ""
    Public Property Nit_imp_exp() As String = ""
    Public Property Clase() As String = ""
    Public Property Mod_transporte() As String = ""
    Public Property Total_liquidar() As Double = 0.0
    Public Property Total_general() As Double = 0.0
    Public Property Codigo_Barra As String = ""



    Sub New()
        '#EJC: Add your constructor here... 
    End Sub

    Public Function Clone() As Object Implements System.ICloneable.Clone
        Return MyBase.MemberwiseClone()
    End Function

End Class

File: WMS.EntityCore/Ticket/clsBeTms_ticket_pol.cs
namespace WMS.EntityCore.Ticket
{
    public class clsBeTms_ticket_pol : ICloneable
    {
        public int IdTicket { get; set; } = 0;
        public int IdOrdenTmsEnc { get; set; } = 0;
        public string NoPoliza { get; set; } = "";
        public string Dua { get; set; } = "";
        public DateTime Fecha_poliza { get; set; } = DateTime.Now;
        public string Pais_procede { get; set; } = "";
        public double Tipo_cambio { get; set; } = 0.0;
        public double Total_valoraduana { get; set; } = 0.0;
        public double Total_bultos_peso { get; set; } = 0.0;
        public double Total_usd { get; set; } = 0.0;
        public double Total_flete { get; set; } = 0.0;
        public double Total_seguro { get; set; } = 0.0;
        public string User_agr { get; set; } = "";
        public DateTime Fec_agr { get; set; } = DateTime.Now;
        public string User_mod { get; set; } = "";
        public DateTime Fec_mod { get; set; } = DateTime.Now;
        public int IdRegimen { get; set; } = 0;
        public string Clave_aduana { get; set; } = "";
        public string Nit_imp_exp { get; set; } = "";
        public string Clase { get; set; } = "";
        public string Mod_transporte { get; set; } = "";
        public double Total_liquidar { get; set; } = 0.0;
        public double Total_general { get; set; } = 0.0;
        public string Codigo_Barra { get; set; } = "";

        public clsBeTms_ticket_pol()
        {
        }

        public object Clone()
        {
            return this.MemberwiseClone();
        }
    }
}